h/static/scripts/directive/test/sidebar-tutorial-test.js

```diff
@@ -9,7 +9,7 @@ describe('SidebarTutorialController', function () {
       var session = {
         state: {
           preferences: {
-            show_sidebar_tutorial: true
+            'show_sidebar_tutorial': true
```

Comment:
Given that this linter rule caught zero instances of us actually violating it and five instances of us interacting with a platform that doesn't by default use lowerCamelCase, perhaps we shouldn't switch it on?